modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java

```diff
@@ -218,6 +221,39 @@ protected UserGroup testDeleteUserGroupUsers_addUserGroup()
 		return _postUserGroup();
 	}
 
+	@Override
+	protected UserGroup testGetAssetLibrariesUserGroupsPage_addUserGroup(
+			Long assetLibraryId, UserGroup userGroup)
+		throws Exception {
+
+		UserGroup userGroup1 = _postUserGroup(userGroup);
```

Comment:
Maybe this should be rename to userGroup instead of userGroup1.

Reply:
You mean reuse the attribute?

Reply:
yes, is it possible?